react-state/my-app/src/ImageCaption.tsx

```diff
@@ -1,11 +1,23 @@
-type Props = {
-  caption: string;
+import { useState } from 'react';
+
+type Prop = {
+  caption: string[];
 };
 
-export function ImageCaption({ caption }: Props) {
+export function ImageCaption({ caption }: Prop) {
```

Comment:
the prop name should be plural since you're passing in an array of captions not just one